microsoft/devhome
# [DevInsights] - Remove colors in Winlogs page so data is visible in dark mode. Also, add filtering using log category.

## Summary of the pull request
1. Remove alternate row color, which made it difficult to read logs in dark mode.
2. Add filtering by adding a dropdown in the header for log categories.

## Validation steps performed
1. Launch PI and attach to process
2. Wait for logs
3. Change to dark theme and make sure all logs are visible
4. filter using the dropdown and make sure the filtered results are correct.

## PR checklist
- [ ] Closes #3061
- [ ] Closes #3050
